app/views/pages/developer-guide/integrations/adding-hcaptcha-spam-protection.liquid

```diff
@@ -332,6 +332,42 @@ document.addEventListener('DOMContentLoaded', function() {
 
 The above code will use an element with the class `my-customized-image` as the trigger for form submission with hCaptcha verification.
 
+## Example 5: Using hCaptcha custom HTML attributes
+
+You can use custom HTML attributes with the spam_protection tag to access additional hCaptcha functionality. Find an example below:
+
+#### app/forms/form_with_captcha.liquid
+
+```liquid
+{% raw %}
+---
+name: form_with_captcha
+resource: model_with_title
+resource_owner: "anyone"
+configuration:
+spam_protection:
+  hcaptcha:
+fields:
+  properties:
+    title:
+return_to: /success_page
+---
+
+{% form html-novalidate: true, html-class: 'model-form' %}
```

Comment:
@andrew-3 @diana-lakatos `{% form %}` tag is deprecated. Instead we should provide example with page that have normal html `<form>`. 
Inside <form> you can use `spam_protection` tag or you can enter own code for generating hcaptcha code (https://github.com/Platform-OS/product-marketplace-template/blob/master/app/views/partials/hcaptcha.liquid)
In order to validate hcaptcha you can use this function https://github.com/Platform-OS/product-marketplace-template/blob/master/app/views/partials/lib/validations/hcaptcha.liquid

Please do not add examples that use `{% form %}` tag. We should change all examples that use it.

Reply:
@diana-lakatos I've created ticket for it some time ago, sadly it was closed.....